modules/database/src/main/java/org/eclipse/xpanse/modules/database/serviceconfiguration/ServiceConfigurationEntity.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * SPDX-FileCopyrightText: Huawei Inc.
+ *
+ */
+
+package org.eclipse.xpanse.modules.database.serviceconfiguration;
+
+import com.fasterxml.jackson.annotation.JsonFormat;
+import io.hypersistence.utils.hibernate.type.json.JsonType;
+import jakarta.persistence.Column;
+import jakarta.persistence.Convert;
+import jakarta.persistence.Entity;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import java.time.OffsetDateTime;
+import java.util.Map;
+import java.util.UUID;
+import lombok.Data;
+import org.eclipse.xpanse.modules.database.common.ObjectJsonConverter;
+import org.hibernate.annotations.Type;
+import org.springframework.format.annotation.DateTimeFormat;
+
+/**
+ * ServiceConfigurationEntity for persistence.
+ */
+@Table(name = "SERVICE_CONFIGURATION")
+@Entity
+@Data
+public class ServiceConfigurationEntity {
+
+    @Id
+    @Column(name = "service_id", nullable = false)
```

Comment:
This table is created as a separate table. Instead, we need this as a child table of DEPLOY_SERVICE with foreign key as SERVICE_ID. (The first requirement in the user story)

something like this https://www.baeldung.com/jpa-one-to-one#spk-strategy